test/models/development_test.rb

```diff
@@ -203,19 +204,81 @@ def development
 
   test "nearby developments" do
     far_dev = developments(:one)
-    far_dev.latitude = 40.000000
+    far_dev.latitude  =  40.000000
```

Comment:
Unnecessary spacing detected.<br>Operator `=` should be surrounded by a single space.